frontend/src/__tests__/App.test.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import App from '../App';
+import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import { store } from '../redux/store';
+
+describe('dispatching received data to redux store on render', () => {
```

Comment:
@manedurphy nice use of describe! Much easier to read and understand more tests in one place